src/Instance/MoveHoldingContext/MoveHoldingContext.js

```diff
@@ -59,7 +63,9 @@ const MoveHoldingContext = ({
     if (!result.destination) return;
     const from = result.source.droppableId;
     const to = result.destination.droppableId;
+    const targetTitle = rightInstance.id === to ? rightInstance.title : leftInstance.title;
     setDragToId(to);
+    setMovingTargetName(targetTitle);
```

Comment:
you don't need this as you already have `to` in state and target name can be found in render